Dockerfile

```diff
@@ -1,19 +1,18 @@
-FROM crystallang/crystal:1.2.2-alpine AS builder
+FROM docker.io/crystallang/crystal:1.6.2-alpine AS builder
 RUN apk update && apk upgrade && apk add sqlite-static
 WORKDIR /build/
 ARG version
 RUN git clone --branch ${version:-dist} --depth 1 https://github.com/toddsundsted/ktistec .
-RUN shards install --production
+RUN shards update && shards install --production
```

Comment:
implemented in https://github.com/toddsundsted/ktistec/commit/648b46397883f79a70b705c20fd3526186662bd0